modules/ROOT/pages/fcct-config.adoc

```diff
@@ -0,0 +1,107 @@
+:experimental:
+= FCCT Configuration and Common Use Cases
+
+== Scope
+This document provides the configuration specification for human-readable FCCT YAML files. Once written, use fcct to convert the YAML file into the machine-readable JSON Ignition file.
```

Comment:
This mentions FCCT and `fcct` without any link, description or acronym expansion.

I'd go for something shorter like:

> This document describes configuration options for Fedora CoreOS nodes, covering both human-friendly and machine-friendly approaches.

We can then further split this page in three sections:
 * YAML, JSON, and transpiling
 * FCC
 * Ignition